outputs/core_pools.json

```diff
@@ -31,28 +31,27 @@
     "0xf16aee6a71af1a9bc8f56975a4c2705ca7a782bc0002000000000000000004bb": "20WETH-80ALCX",
     "0x8353157092ed8be69a9df8f95af097bbf33cb2af0000000000000000000005d9": "GHO/USDT/USDC",
     "0x3ff3a210e57cfe679d9ad1e9ba6453a716c56a2e0002000000000000000005d5": "STG/USDC",
-    "0xcf7b51ce5755513d4be016b0e28d6edeffa1d52a000200000000000000000617": "80RDNT-20wETH"
+    "0xcf7b51ce5755513d4be016b0e28d6edeffa1d52a000200000000000000000617": "80RDNT-20wETH",
+    "0xff42a9af956617e4c3532ef2fc7567465efe4909000200000000000000000672": "50KEP/50rsETH"
   },
   "polygon": {
     "0xcd78a20c597e367a4e478a2411ceb790604d7c8f000000000000000000000c22": "maticX-WMATIC-BPT",
     "0xee278d943584dd8640eaf4cc6c7a5c80c0073e85000200000000000000000bc7": "ECLP-WMATIC-MATICX",
     "0xf0ad209e2e969eaaa8c882aac71f02d8a047d5c2000200000000000000000b49": "ECLP-WMATIC-stMATIC"
   },
   "arbitrum": {
-    "0x0c8972437a38b389ec83d1e666b69b8a4fcf8bfd00000000000000000000049e": "wstETH/rETH/sfrxETH",
     "0x125bc5a031b2db6733bfa35d914ffa428095978b000200000000000000000514": "ECLP-AUSDC-AUSDT",
     "0x2ce4457acac29da4736ae6f5cd9f583a6b335c270000000000000000000004dc": "sFRAX/4POOL",
     "0x2d6ced12420a9af5a83765a8c48be2afcd1a8feb000000000000000000000500": "cbETH/rETH/wstETH",
     "0x3fd4954a851ead144c2ff72b1f5a38ea5976bd54000000000000000000000480": "ankrETH/wstETH-BPT",
     "0x451b0afd69ace11ec0ac339033d54d2543b088a80000000000000000000004d5": "plsRDNT-Stable",
-    "0x4a2f6ae7f3e5d715689530873ec35593dc28951b000000000000000000000481": "wstETH/rETH/cbETH",
     "0x9791d590788598535278552eecd4b211bfc790cb000000000000000000000498": "wstETH-WETH-BPT",
-    "0xade4a71bb62bec25154cfc7e6ff49a513b491e81000000000000000000000497": "rETH-WETH-BPT",
     "0xb61371ab661b1acec81c699854d2f911070c059e000000000000000000000516": "ezETH/wstETH",
     "0xc2598280bfea1fe18dfcabd21c7165c40c6859d30000000000000000000004f3": "wstETH/sfrxETH",
     "0xd0ec47c54ca5e20aaae4616c25c825c7f48d40690000000000000000000004ef": "rETH/wETH BPT",
     "0x423a1323c871abc9d89eb06855bf5347048fc4a5000000000000000000000496": "4POOL",
-    "0x32df62dc3aed2cd6224193052ce665dc181658410002000000000000000003bd": "RDNT-WETH"
+    "0x32df62dc3aed2cd6224193052ce665dc181658410002000000000000000003bd": "RDNT-WETH",
+    "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH"
```

Comment:
What's this guy about?

Reply:
it should be in there according to the bip: https://forum.balancer.fi/t/bip-493-enable-ankr-ankreth-gauge-arbitrum/5363

Reply:
it is on the whitelist because it is yield fee exempt